_includes/v22.2/orchestration/kubernetes-limitations.md

```diff
@@ -26,4 +26,6 @@ When starting Kubernetes, select machines with at least **4 vCPUs** and **16 GiB
 
 #### Storage
 
-At this time, orchestrations of CockroachDB with Kubernetes use external persistent volumes that are often replicated by the provider. Because CockroachDB already replicates data automatically, this additional layer of replication is unnecessary and can negatively impact performance. High-performance use cases on a private Kubernetes cluster may want to consider using [local volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local).
+- At this time, orchestrations of CockroachDB with Kubernetes use external persistent volumes that are often replicated by the provider. Because CockroachDB already replicates data automatically, this additional layer of replication is unnecessary and can negatively impact performance. High-performance use cases on a private Kubernetes cluster may want to consider using [local volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local).
+
```

Comment:
This needs a heading; it doesn't belong under "Storage"

Reply:
You're right, I thought about that just as I was drifting off to sleep last night: "Wait, why did I put it under Storage?" I'll fix it.

Reply:
Done